tests/qa.py

```diff
@@ -57,12 +89,13 @@ def test_qa_sign_tx_with_token(cmd):
             TxInput(fake.sha256(True), 0, path),
             ]
     outputs = [
-            TxOutput(fake.pyint(1), fake_script(), 1),
+            TxOutput(1, fake_script(), 1, True),
+            TxOutput(2, fake_script(), 1, True),
             TxOutput(fake.pyint(1), fake_script()),
             ]
-    tx = Transaction(1, [token.uid], inputs, outputs)
-    print("QA::sign_tx_with_token::token:", str(token))
-    print("QA::sign_tx_with_token::tx:", str(tx))
+    tx = Transaction(0, 1, [token.uid], inputs, outputs)
```

Comment:
What's this new first parameter?

Reply:
This qa document was not updated on #67 where signal_bits was introduced so it was updated here.

The idea of the qa would be a form of manual validation for the UI, so the qa assignee can check that everything is being shown as intended on the screen.